.github/workflows/release.yaml

```diff
@@ -119,7 +119,7 @@ jobs:
         run: echo LAST_COMMIT_MSG=$(git --no-pager log -1 --pretty=%B) >> $GITHUB_ENV
 
       - name: Decides if Docs should be deployed
-        if: startsWith(env.LAST_COMMIT_MSG, 'ci(changesets):') && env.RELEASE_VERSION_HIGHER_THAN_LATEST == 'true'
+        if: startsWith(env.LAST_COMMIT_MSG, 'ci:') && env.RELEASE_VERSION_HIGHER_THAN_LATEST == 'true'
```

Comment:
Could this cause an issue?

Should we look for `ci: release` or `ci: releasing` to be sure it's the release PR?

Reply:
How come?

I changed from `ci(changesets):` to `ci:` because I removed the `(changesets)` portion of the titles.

Reply:
cc @nedsalk CI 🧙

Reply:
I believe the `ci(changesets):` was used to only identify the changeset merge commit, so we want to identify the follow [commit](https://github.com/FuelLabs/fuels-ts/pull/2804/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR73) message...

What happens if you merge a pull review `ci: not release` - will this not deploy the docs?

@nedsalk will for sure know more than me :)

Reply:
@nedsalk Just to clarify, does the thumbs-up reaction indicate that @petertonysmith94's concerns are correct?

Reply:
Haha, I tried with two emojis to indicate it's correct. Yes, @petertonysmith94 is right here. 
One can interpret it as me hitting the like and hooray for his last sentence, though :smile:  
> @nedsalk will for sure know more than me :)

Reply:
[ae2046c](https://github.com/FuelLabs/fuels-ts/pull/2804/commits/ae2046c16ccea8e6ca147d5696eb9e4373597f4d)

Reply:
@petertonysmith94 IIRC, we have been re-building the docs every time a new PR goes to master (nightly).

For this to happen, I think we need the conditional to be as I did it:
 - `startsWith(env.LAST_COMMIT_MSG, 'ci:')`

This modification seems to be changing the behavior.

Am I missing something?

---

Before:
 1. Condition:
    - `startsWith(env.LAST_COMMIT_MSG, 'ci(changesets):')`
 1. Matches:
    - `"ci(changesets): versioning packages - ${{ env.RELEASE_VERSION }} @ ${{ github.ref_name }}"`
    - `"ci(changesets): versioning packages"`

After:
 1. Condition:
    - `env.LAST_COMMIT_MSG == 'ci: versioning packages and changesets'`
 1. Matches:
    - `ci: versioning packages and changesets`
 1. Misses:
    - `ci: releasing ${{ env.RELEASE_VERSION }} @ ${{ github.ref_name }}`

Reply:
@arboleya that does seem like a problem - my mistake. 

With the following conditional:
- `startsWith(env.LAST_COMMIT_MSG, 'ci:')`

Would it not also pick up the merge of all `ci:` branches merged into master? For example:
- https://github.com/FuelLabs/fuels-ts/pull/2887

Maybe this isn't an issue, and I'm over thinking.

Reply:
I see, then a prefix may be good after all.

What do you think? https://github.com/FuelLabs/fuels-ts/pull/2804/commits/80f312b315bac262277a1810f1ac35c5cc4a80e9